src/aspire/basis/ffb_2d.py

```diff
@@ -197,11 +197,10 @@ def _evaluate_t(self, x):
         freqs = np.reshape(self._precomp["freqs"], (2, n_r * n_theta))
 
         # number of 2D image samples
-        n_images = x.n_images
-        x_data = x.data
+        n_images = x.shape[0]
```

Comment:
I think the docstring `:return:` should say "The evaluation of the coefficient array `x`..."